admin/css/metisMenu.css

```
@@ -0,0 +1,131 @@
+    height: .5em; */
+    /* border-width: 1px 0 0 1px; */
+    /* border-style: solid;
+    border-color: currentColor;
+    border-color: initial;
+    right: 1em; */
+    transform: rotate(-90deg) translate(0, -50%);
+    transform-origin: top;
+    top: 50%;
+    transition: all .3s ease-out;
+    right: 40px;
+     content: "\e64b";
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

content is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)